sdks/python/apache_beam/io/tfrecordio.py

```diff
@@ -178,6 +180,44 @@ def read_records(self, file_name, offset_range_tracker):
           yield self._coder.decode(record)
 
 
+def _create_tfrcordio_source(
+    file_pattern=None, coder=None, compression_type=None,
+    validate=False):
```

Comment:
<!--new_thread; commit:d4254d9fe817a78ccd0f7704c87bc89f903add8c; resolved:0-->
validate is always False? I expected it to be passed below or from ReadAllFiles.

Reply:
We intentionally disable validation for ReadAllFrom pattern since we are reading a PCollection of globs and some of the globs might be empty (which result in validation errors for ReadFrom pattern). Added a comment